src/main/java/org/example/game2048/movements/Movement.java

```diff
@@ -0,0 +1,172 @@
+package org.example.game2048.movements;
+import org.example.game2048.point.Point;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Movement {
```

Comment:
znacznie lepiej teraz to wyglada ! 
powiedz mi tylko czemu wszystkie metody i zmienne sa `static`? tego troche nie rozumiem :P gdybys odpalil 2 gry na raz kazda gra miala by te same zmienne.